rhel6/templates/template_OVAL_service_disabled

```diff
@@ -1,98 +0,0 @@
-<def-group>
-  <definition class="compliance" id="service_%SERVICENAME%_disabled"
-  version="1">
-    <metadata>
-      <title>Service %SERVICENAME% Disabled</title>
-      <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
-      </affected>
-      <description>The %SERVICENAME% service should be disabled if possible.</description>
-    </metadata>
-   <criteria comment="package %PACKAGENAME% removed or service %SERVICENAME% is not configured to start" operator="OR">
-    <extend_definition comment="%PACKAGENAME% removed" definition_ref="package_%PACKAGENAME%_removed" />
-    <criteria operator="AND" comment="service %SERVICENAME% is not configured to start">
-      <criterion comment="%SERVICENAME% runlevel 0" test_ref="test_runlevel0_%SERVICENAME%" />
```

Comment:
I see a discrepancy between this line of the RHEL6 template and [the new template template](https://github.com/OpenSCAP/scap-security-guide/pull/2850/files?utf8=%E2%9C%93&diff=split#diff-c71262d467c8b535cfbfd7b53f8eee36R51) (the `_off` suffix is there, but not here).

Reply:
yeah, that was necessary because of conflicts with some hardcoded fixes. probably should have mentioned it here.

Reply:
OVAL has this idea that objects that are the same can be reused. In practice this has backfired many many times for us. We even created warnings and triggers in the build system for it. I think we are best off if the object state and test are all defined in one file and only used in that one file. It can be a bit wasteful in the end but it prevents hard to find bugs.

Reply:
OK, good to know.